native/cocos/math/Vec3.cpp

```diff
@@ -27,6 +27,36 @@
 #include "math/MathUtil.h"
 #include "math/Quaternion.h"
 
+#if (CC_PLATFORM == CC_PLATFORM_ANDROID)
+    #include <cpu-features.h>
+#endif
+
+#if (CC_PLATFORM == CC_PLATFORM_IOS)
+    #if defined(__arm64__)
+        #define USE_NEON64
+        #define INCLUDE_NEON64
+    #elif defined(__ARM_NEON__)
+        #define USE_NEON32
+        #define INCLUDE_NEON32
+    #else
+    #endif
+#elif (CC_PLATFORM == CC_PLATFORM_ANDROID)
+    #if defined(__arm64__) || defined(__aarch64__)
+        #define USE_NEON64
+        #define INCLUDE_NEON64
+    #elif defined(__ARM_NEON__)
+        #define INCLUDE_NEON32
+    #endif
+#endif
+
+#if defined(USE_NEON64) || defined(USE_NEON32) || defined(INCLUDE_NEON32)
```

Comment:
What's the difference between `USE_NEON32 ` and `INCLUDE_NEON32 `?

Reply:
It's the old logic in MathUtil.

When iOS 32 bits support is not removed, USE_NEON32 and INCLUDE_NEON32 is both enabled
On Android, only INCLUDE_NEON32 is enabled, USE_NEON32 is disabled.